src/Scan.js

```diff
@@ -705,20 +736,29 @@ export class Scan extends React.Component {
   }
 
   fetchItems = async (barcode, offset = 0) => {
-    const { mutator } = this.props;
+    const {
+      mutator,
+      stripes,
+    } = this.props;
     const { selectedBarcode } = this.state;
     const itemBarcode = barcode || selectedBarcode;
     const query = `barcode==${itemBarcode}`;
 
     this.setState({
       checkedinItem: null,
     });
-    mutator.items.reset();
 
+    const mutatorFunction = getMutatorFunction(stripes, mutator);
+
+    if (isEmpty(mutatorFunction)) {
+      return undefined;
```

Comment:
The `fetchItems` must return an array, otherwise the condition `checkedinItems.length > 1` will cause the page to crash.

Reply:
Fixed